aiida/tools/importexport/archive/readers.py

```diff
@@ -332,7 +332,7 @@ def get_repository_container(self) -> Container:
         """Return an instance mapped to the repository container."""
         self._extract(path_prefix='')
         assert self._sandbox is not None  # required by mypy
-        return Container(Path(self._sandbox.abspath) / 'container')
+        return Container(str(Path(self._sandbox.abspath) / 'container'))
```

Comment:
Has `disk-objectstore` removed support for `pathlib.Path` here? If so, why? If not, why convert to string?

Reply:
No, just a typing faux pas that I've added a "fix" for the next release: https://github.com/aiidateam/disk-objectstore/commit/c0120568a992b41a55b325f3217d4902b5281070

Can remove the `str` if you want, and would just have to add a type ignore for now

Reply:
That might be preferable because as soon as the fix is released in `disk-objectstore` then `mypy` would complain about the unused type ignore, right? That would ensure we remove it when no longer necessary whereas the `str` conversion may hang around for a long time

Reply:
done 👍